### DISCUSSION TITLE
DHT22

### DISCUSSION THREAD

**Discussion by o0shojo0o** [pixelit-project/PixelIt#30] [Node-RED Flows]
![image](https://user-images.githubusercontent.com/19333515/121035324-0cec9400-c7ae-11eb-99f9-08636dcf406d.png)
~~~
[{"id":"8a0f7072.1ea15","type":"http request","z":"eeacac8e.84e57","name":"Get DHT22","method":"GET","ret":"obj","paytoqs":false,"url":"http://192.168.0.137/api/dhtsensor","tls":"","persist":false,"proxy":"","authType":"","x":510,"y":260,"wires":[["61e90071.7806c"]]},{"id":"aa547419.b0b348","type":"inject","z":"eeacac8e.84e57","name":"Every 3s","topic":"","payload":"","payloadType":"date","repeat":"3","crontab":"","once":true,"onceDelay":"0.1","x":120,"y":260,"wires":[["69795a92.db2494"]]},{"id":"69795a92.db2494","type":"is online","z":"eeacac8e.84e57","name":"check PIXEL IT is online","url":"192.168.0.137","action":"1","x":310,"y":260,"wires":[["8a0f7072.1ea15"]]},{"id":"ecef3ff1.58ac3","type":"Screen Init","z":"eeacac8e.84e57","name":"Screen Init","inscreenName":"dht22_temp","induration":"5","x":850,"y":240,"wires":[["7d7137f9.c44e38"]]},{"id":"ec6f3c04.75503","type":"Text","z":"eeacac8e.84e57","name":"Text","intextString":"{{temp}}","inbigFont":"false","inscrollText":"false","inscrollTextDelay":"0","incenterText":"true","inposX":"7","inposY":"1","incolorR":"255","incolorG":"255","incolorB":"255","x":1130,"y":240,"wires":[["8ec2d42c.71a188"]]},{"id":"7d7137f9.c44e38","type":"Bitmap","z":"eeacac8e.84e57","name":"Bitmap","indata":"324","inposX":"0","inposY":"0","inwidth":"8","inheight":"8","x":1000,"y":240,"wires":[["ec6f3c04.75503"]]},{"id":"61e90071.7806c","type":"function","z":"eeacac8e.84e57","name":"Text format","func":"msg.temp = msg.payload.temperature + \"°C\"\nmsg.hum = msg.payload.humidity + \"%\"\nreturn msg;","outputs":1,"noerr":0,"x":670,"y":260,"wires":[["ecef3ff1.58ac3","7dd2972.e7b3468"]]},{"id":"7dd2972.e7b3468","type":"Screen Init","z":"eeacac8e.84e57","name":"Screen Init","inscreenName":"dht22_hum","induration":"5","x":850,"y":280,"wires":[["964f3594.ca53a8"]]},{"id":"4b16c65a.faa1b8","type":"Text","z":"eeacac8e.84e57","name":"Text","intextString":"{{hum}}","inbigFont":"false","inscrollText":"false","inscrollTextDelay":"0","incenterText":"true","inposX":"7","inposY":"1","incolorR":"255","incolorG":"255","incolorB":"255","x":1130,"y":280,"wires":[["8ec2d42c.71a188"]]},{"id":"964f3594.ca53a8","type":"Bitmap","z":"eeacac8e.84e57","name":"Bitmap","indata":"66","inposX":"0","inposY":"0","inwidth":"8","inheight":"8","x":1000,"y":280,"wires":[["4b16c65a.faa1b8"]]},{"id":"8ec2d42c.71a188","type":"link out","z":"eeacac8e.84e57","name":"Weather Stuhr [weather_stuhr] Link","links":["df7f7c66.5d7ce","73e9236a.c7c1ec"],"x":1235,"y":260,"wires":[]}]
~~~
Dependency: https://flows.nodered.org/node/node-red-contrib-isonline

**Comment by Techposts** [comment]
DHT isn't working. Please help.
![chrome_oby1NrgEx2](https://user-images.githubusercontent.com/28442414/159174895-c53d3d47-5d12-4e35-b566-5ec4c5dd6bb6.jpg)
![chrome_XGBrMXGo9n](https://user-images.githubusercontent.com/28442414/159174896-52ab414f-6ebc-4377-b9e8-03afacd7a4a8.jpg)

**Reply by o0shojo0o** [reply]
this node is missing  https://flows.nodered.org/node/node-red-contrib-isonline

**Reply by Techposts** [reply]
Thanks! But I still can't get the data on Pixelit. It  isn't showing the Temp or hum.
![chrome_GfVEZRX0jc](https://user-images.githubusercontent.com/28442414/159293926-4866a78e-e3e7-4985-b3a7-c29ac8ae0091.jpg)
![chrome_V8P64OxhBG](https://user-images.githubusercontent.com/28442414/159293939-14e65a1a-5695-4c62-b811-73ae095bc980.jpg)
![chrome_15wSHn4Ixv](https://user-images.githubusercontent.com/28442414/159293942-a0a3b8f2-1d39-4329-840a-f58c7a129b6e.jpg)

**Reply by o0shojo0o** [reply]
Did you add these screens to the playlist as well?

**Reply by o0shojo0o** [reply]
and these must be connected to "Screen Data Update"

**Reply by Techposts** [reply]
I have added them to Screen Data Update. Still no success
And I read the Documentation, couldn't understand exactly what's the use for Playlist.
![image](https://user-images.githubusercontent.com/28442414/159337768-4675ab59-b378-4c76-b342-bdffd0c0c318.png)

**Reply by Techposts** [reply]
Does it have to do anything from the Pixelit settings?
![image](https://user-images.githubusercontent.com/28442414/159338260-2c23030c-eeb6-4430-a6f5-c5a3a03ea666.png)

**Reply by o0shojo0o** [reply]
Here in the example flow is the "Display Playlist" inside, it describes which screens should be displayed.
https://docs.bastelbunker.de/pixelit/tools.html#einsteiger-flow

**Reply by o0shojo0o** [reply]
![image](https://user-images.githubusercontent.com/19333515/159339729-f104acbf-7761-49cb-944c-97916fd8ec8c.png)

**Reply by Techposts** [reply]
I understood what you said. It's fixed now. Had to update the Playlist screens. Thanks for the quick response and help. Really appreciate it. I will spread the word through my channel. Awesome work.

**Reply by o0shojo0o** [reply]
Thank you :)

**Reply by Techposts** [reply]
Youtube, Internal clock DHT22_temp, Instagram followers, is showing but Humidity is not displayed. It displays Instagram followers and then temp. Skips humidity data
[{"screenName":"internal_clock"},{"screenName":"youtube"},{"screenName":"instagram"},{"screenName":"dht22_hum"},{"screenName":"dht22_temp"}]

![image](https://user-images.githubusercontent.com/28442414/159344775-e803ebfd-c3d0-4782-b028-1f12b3e40133.png)

**Reply by Techposts** [reply]
Fixed. Had to create two different flow for each
![image](https://user-images.githubusercontent.com/28442414/159345168-529406a7-d552-42a4-8f67-9d1172e93bda.png)
